test/Driver/Dependencies/one-way-external.swift

```diff
@@ -21,7 +21,7 @@
 // RUN: touch -t 201401240005 %t/*
 // RUN: touch -t 201401240006 %t/*.o
 // RUN: touch -t 201401240004 %t/*-external
-// RUN: touch -t 300004010005 %t/other1-external
+// RUN: touch -t 203704010005 %t/other1-external
```

Comment:
Mm. I'd prefer if the ones not related to linking were separate. (The driver one is clearly related.) They could go in your armv6 PR, or just be small separate PRs (or one separate PR) that we'd just take as general goodness.